### DISCUSSION TITLE
MediaMTX and RPI Camera Config

### DISCUSSION THREAD

**Discussion by PrestonV78** [bluenviron/mediamtx#3989] [Questions]
### Question

I am trying to use my RPi 4 and Arducam 5MP OV5647 camera to get a better view in my P1S

I was able to get it all set up and running MediaMTX to stream video, but how I think MediaMTX has settings messing w the video.

The video doesn't look like it's 1080p like the camera suggests and I need to rotate the video 90° if possible (can do after the fact I guess).

How would I make changes to the aspect ratio and such to get these changes?

**Comment by aler9** [comment]
This is impossible to replicate if you don't provide server version, server logs, configuration file.
If you still have problems, open another discussion or a bug report.
